engine/flink/avro-util/src/main/scala/pl/touk/nussknacker/engine/avro/schemaregistry/confluent/client/CachedConfluentSchemaRegistryClient.scala

```diff
@@ -10,18 +10,17 @@ import pl.touk.nussknacker.engine.avro.schemaregistry.SchemaRegistryError
 import pl.touk.nussknacker.engine.kafka.KafkaConfig
 import pl.touk.nussknacker.engine.util.cache.Cache
 
-import scala.concurrent.duration.Duration
-
 /**
   * We use there own cache engine because ConfluentCachedClient doesn't cache getLatestSchemaMetadata and getSchemaMetadata
   */
-class CachedConfluentSchemaRegistryClient(val client: CSchemaRegistryClient, schemaCache: Cache[Schema], latestSchemaTtl: Option[Duration]) extends ConfluentSchemaRegistryClient with LazyLogging {
+class CachedConfluentSchemaRegistryClient(val client: CSchemaRegistryClient, schemaCache: Cache[Schema], latestSchemaCache: Cache[Schema])
+  extends ConfluentSchemaRegistryClient with LazyLogging {
 
   private val latestCacheNamespace = "latest"
 
   override def getLatestSchema(subject: String): Validated[SchemaRegistryError, Schema] = {
     handleClientError {
-      schemaCache.getOrCreate(s"$subject-$latestCacheNamespace", latestSchemaTtl) {
+      latestSchemaCache.getOrCreate(s"$subject-$latestCacheNamespace") {
```

Comment:
you don't need this trick with `latestCacheNamespace` anymore - you can just pass `subject`

Reply:
Done.